extensions/amp-story/1.0/amp-story-consent.css

```diff
@@ -0,0 +1,218 @@
+/**
```

Comment:
overall: there are a lot of `calc`'s in here; is it possible to remove any?  probably not a huge deal, but `calc` can be less performant.

Reply:
I removed the first two ones, but kept one on desktop that makes the positioning a lot easier